wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md

```diff
@@ -0,0 +1,73 @@
+---
+tags:
+    - BSC
+    - beatmap spotlight curator
+---
+
+# Beatmap Spotlight Curators
+
+The **Beatmap Spotlight Curators** is a group that assists in selecting high-quality maps for each season of the [Beatmap Spotlights](/wiki/Beatmap_Spotlights).
+
+## Becoming a curator
+
+In most cases, new curators are suggested by the respective game mode team to fit a certain niche. Such suggestions are evaluated by the project managers, team leaders, and curators themselves.
+
+In addition, there may be application cycles for certain game modes announced for certain seasons.
+
+All applicants are assessed based on their proficiency as a player, mapper, modder, and tournament mappool selector for their respective game mode. Applicants do not need to be proficient in all of these things to be selected, but having broad experience helps. The goal is to have a roster of curators with a mix of various skill levels, experiences, and proficiencies. However, the number of curators is currently kept small and limited while the project is developing.
+
+The [Beatmap Spotlight Curator(s) group page](https://osu.ppy.sh/groups/48) lists all of the team members. A more detailed member listing for all of the various roles within the Beatmap Spotlight Curators can be found on the respective [Beatmap Spotlight page](/wiki/Beatmap_Spotlights).
```

Comment:
delete

Reply:
Just to make sure that this is in reference to the entire `Becoming a Curator`, portion with the section about the group page, or is there a more specific portion that you are referencing to delete. Imo the comment is not as specific owo

Reply:
it's kinda in the wrong section. at the listing or intro fits better